src/android/AssetBundle.java

```diff
@@ -191,7 +191,7 @@ public String getRootUrlString() {
             JSONObject runtimeConfig = getRuntimeConfig();
             if (runtimeConfig != null) {
                 try {
-                    rootUrlString = runtimeConfig.getString("ROOT_URL");
+                    rootUrlString = runtimeConfig.getString("ROOT_URL").replace("localhost","10.0.2.2");
```

Comment:
A few suggestions:
1 - Could you split this operation into two lines (not replacing directly after `getString`) to improve readability;
2 - Extract these string `localhost` and `10.0.2.2` for constants then we can have a readable name for both;

Reply:
Yep, HCP doesn't work in Android emulator without this. That's the only purpose of it, is to make HCP work in Android emulator.

The proposed change is a straight steal from this existing meteor code, which is solving the same problem:
https://github.com/meteor/meteor/blob/3051150f2f5ae953f391802e73682fba613b3d46/packages/boilerplate-generator/template-web.cordova.js#L42-L50

I can split the lines for readability and extract the strings as constants, but I don't know what the test would be for whether it's dev or not. Is there a standard test for that?

Reply:
hm ok, if Meteor is already doing the same I believe this is safe. Please just update your code to have constants and let's move this forward.

Reply:
cool  -- I extracted the constants and split the line into two.